apps/condo/domains/ticket/components/TicketId/TicketFileList.tsx

```diff
@@ -49,8 +53,19 @@ export const TicketFileList: React.FC<ITicketFileListProps> = ({ files }) => {
     })), [files])
 
     const downloadFile = useCallback(async (file: UploadFile) => {
-        const response = await fetch(file.url)
-        const blob = await response.blob()
+        const redirectResponse = await fetch(file.url, {
+            credentials: 'include',
+            headers: {
+                '-no-redirect': 'true',
```

Comment:
'--no-redirect' header confuses with logic below, that actually utilizes redirectUrl. The same confusion is on server side, — "if redirect, then respond with redirect in different form". Maybe to rename the header to "shallow-redirect" and to write comments about why this kind of behavior was implemented?